k9mail/src/main/java/com/fsck/k9/preferences/AccountSettings.java

```diff
@@ -53,10 +53,6 @@
         s.put("chipColor", Settings.versions(
                 new V(1, new ColorSetting(0xFF0000FF))
             ));
-        s.put("cryptoApp", Settings.versions(
-                new V(1, new StringSetting("apg")),
```

Comment:
Does setting this to null remove the setting? I think it does, but this is never done anywhere and the code that deals with this isn't very simple :\